documentation/release_notes.markdown

```diff
@@ -4,87 +4,271 @@ title: "Puppet Server: Release Notes"
 canonical: "/puppetserver/latest/release_notes.html"
 ---
 
+## Puppet Server 1.1.0
```

Comment:
I added back the 2.1 notes, which I swear I thought I merged the first time around, but eh...

It's one big file now with all notes ever, waiting to hear about teasing them apart, which ideally we could do outside of the merge commit.